exercises/G-conditionals-2/exercise-4.js

```diff
@@ -8,7 +8,12 @@
   Hint: Google how to check if a string contains a word
 */
 
-function containsCode(sentence) {}
+function containsCode(sentence) {
+    if (sentence.includes("code")) {
```

Comment:
The logic here is correct, but to make this more concise you could have 
```
function containsCode(sentence) {
     return sentence.includes("code")
}
```
This should work the same